docs/snippets/compare_snippet_output.py

```diff
@@ -286,7 +298,7 @@ def check_non_empty_rrd(path: str) -> None:
     from pathlib import Path
 
     assert Path(path).stat().st_size > 0
-    print(f"Confirmed output written to {Path(path).absolute()}")
+    # print(f"Confirmed output written to {Path(path).absolute()}")
```

Comment:
dead code?

Reply:
Yes, but may be useful when debugging later